Scoop/NetworkManager.swift

```diff
@@ -48,5 +47,76 @@ class NetworkManager {
         }
     }
     
+    func getAllRides(completion: @escaping(RideResponse) -> Void) {
+        let endpoint = "\(NetworkManager.hostEndpoint)/api/rides/"
+        
+        AF.request(endpoint, method: .get).validate().responseData { response in
+        switch (response.result) {
+        case .success(let data):
+            let jsonDecoder = JSONDecoder()
+            jsonDecoder.dateDecodingStrategy = .iso8601
+            jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
+            if let ridesResponse = try? jsonDecoder.decode(RideResponse.self, from: data) {
+                completion(ridesResponse)
+            } else {
+                print("Failed to decode getAllRides from JSON")
+            }
+        case .failure(let error):
+            print(error.localizedDescription)
+            }
+        }
+    }
     
+    func getSpecificRide(rideID: Int, completion: @escaping(Ride) -> Void) {
+        let endpoint = "\(NetworkManager.hostEndpoint)/api/ride/\(rideID)/"
+        
+        AF.request(endpoint, method: .get).validate().responseData { response in
```

Comment:
Realized just now that my indenting is off for some of these networking functions, will fix that.